.github/workflows/pull_request_title_lint.yaml

```diff
@@ -0,0 +1,60 @@
+name: "Lint Pull Request Titles"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
+        id: lint_pr_title
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed (newline-delimited).
+          # Default: https://github.com/commitizen/conventional-commit-types
+          types: |
+            fix
+            feat
+            chore
+            ci
+            docs
+            refactor
+            release
+            test
+            revert
```

Comment:
Can you align the types with what we have in the contribution guide? https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md#commit-messages